runtime/doc/cmdline.txt

```diff
@@ -743,7 +743,7 @@ function expand() |expand()|.
 	#<n	(where n is a number > 0) is replaced with old	  *:_#<* *c_#<*
 		file name n.  See |:oldfiles| or |v:oldfiles| to get the
 		number.							*E809*
-		{only when compiled with the |+eval| and |+viminfo| features}
+		{only when compiled with the |+eval| and |+shada| features}
```

Comment:
I thought this is out of the scope of this PR.

Note that now `has('viminfo')` will return zero, but `has('shada')` will return one. This is intentional because even though `:[rw]viminfo` and `'viminfo'` options are kept for compatibility (I think that most of plugin authors do not bother checking for such basic feature should they use it) there are too much differences between old viminfo files and new ShaDa.